ui/src/extensionVariables.ts

```diff
@@ -32,7 +32,9 @@ class UninitializedExtensionVariables implements UIExtensionVariables {
  * Container for common extension variables used throughout the UI package. They must be initialized with registerUIExtensionVariables
  */
 export let ext: UIExtensionVariables = new UninitializedExtensionVariables();
+export let extInitialized: boolean = false;
```

Comment:
We already set `ext` to `UninitializedExtensionVariables` the line above this, which means you will get the following error if you try to use `ext` before you're supposed to: https://github.com/Microsoft/vscode-azuretools/blob/master/ui/src/extensionVariables.ts#L12

Your work with `extInitialized` feels redundant